shared/utils/combine-remediations.py

```diff
@@ -375,79 +375,84 @@ def main():
-                        # corresponding XCCDF <sub> elements
-                        expand_xccdf_subs(fix, remediation_type, remediation_functions)
-                else:
-                    sys.stderr.write("Skipping '%s' remediation script. "
-                                     "The platform identifier in the script is "
-                                     "missing!\n" % (filename))
+                            mod_file += line
+
+                    complexity = None
+                    disruption = None
+                    reboot = None
+                    script_platform = None
+                    strategy = None
+
+                    if 'complexity' in config:
+                        complexity = config['complexity']
+                    if 'disruption' in config:
+                        disruption = config['disruption']
+                    if 'platform' in config:
+                        script_platform = config['platform']
+                    if 'complexity' in config:
+                        reboot = config['reboot']
+                    if 'complexity' in config:
+                        strategy = config['strategy']
+
+                    if script_platform:
+                        product_name, result = fix_is_applicable_for_product(script_platform, product)
+                        if result:
+                            if fixname in fixes:
+                                fix = fixes[fixname]
+                                for child in list(fix):
+                                    fix.remove(child)
+                            else:
+                                fix = etree.SubElement(fixgroup, "fix")
+                                fix.set("rule", fixname)
+                                if complexity is not None:
+                                    fix.set("complexity", complexity)
+                                if disruption is not None:
+                                    fix.set("disruption", disruption)
+                                if reboot is not None:
+                                    fix.set("reboot", reboot)
+                                if strategy is not None:
+                                    fix.set("strategy", strategy)
+                                fixes[fixname] = fix
+                                included_fixes_count += 1
+
+                            fix.text = mod_file
+
+                            # Expand shell variables and remediation functions into
+                            # corresponding XCCDF <sub> elements
+                            expand_xccdf_subs(fix, remediation_type, remediation_functions)
+                    else:
+                        sys.stderr.write("Skipping '%s' remediation script. "
+                                         "The platform identifier in the script is "
+                                         "missing!\n" % (filename))
+        except OSError as e:
+            print("Not merging remediation scripts from the "
```

Comment:
Check for e.errno == errno.ENOENT, else reraise

See https://docs.python.org/2/library/errno.html#module-errno